client/components/ProductOverview/ImageGallery.jsx

```diff
@@ -36,8 +36,8 @@ const ImageGallery = ({ currentStyle, imageIndex, updateImageIndex }) => {
     <div className="imageGalleryContainer">
       <div className="mainImageContainer" style={{ backgroundImage: `url(${currentPhoto})` }} />
       <ThumbnailView photos={photos} handleClick={handleThumbNailClick} className="thumbnailComponent" currentIndex={imageIndex} />
-      <AiOutlineArrowLeft className="leftArrow" onClick={() => handleArrow('left')} size={arrowSize} />
-      <AiOutlineArrowRight className="rightArrow" onClick={() => handleArrow('right')} size={arrowSize} />
+      {imageIndex === 0 ? null : <AiOutlineArrowLeft className="leftArrow" onClick={() => handleArrow('left')} size={arrowSize} />}
```

Comment:
I'm wondering if for these instead of a ternary would the `&&` work like ```imageIndex !== 0 && <AiOUtline... />```

Reply:
Perhaps, what would be the benefits of this?

Reply:
Reduce complexity. You could get rid of `null` and if image index doesn't equal zero, then render arrow left.